Assets/BezierCurve.cs

```diff
@@ -0,0 +1,76 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BezierCurve : MonoBehaviour
```

Comment:
It would be nice if you sorted your assets based on their type. As you did with `Scenes` folder, but for others too. That is, created `Materials`, `Scripts`, and `Prefabs` folders.